lib/core/repository/cms/attribute_builder.rb

```diff
@@ -12,23 +12,21 @@ def self.build(response)
       end
 
       def attributes
-        attributes = response.body
+        attrs = response.body
```

Comment:
Ok, I understand that is heavily used, but it is mutating the same object, so the names could be the same without any problem. I would return to attributes as before because the code becomes better to read.